docs/user_guide/hydromt_cli.rst

```diff
@@ -0,0 +1,295 @@
+It has one mandatory argument:
+
+- `EXPORT_DEST_PATH`: Absolute or relative path to the output folder of the exported data.
+
+The input data catalogs are specified using the `-d, \-\-data` option as in the `build` or `update` commands.
+You are also free to specify several catalogs.
+
+There are two ways to specify the sources/extent of the data to export: either fully from the command line or by using a configuration file.
+
+If you are using the command line, the main options are:
+
+- `-d, \-\-data`: Relative or absolute path to the local yaml data catalog file or name of a predefined data catalog. The data catalog is a yaml file that contains the paths to the data that will be used to export the data. In order to use several catalogs, you can use the `-d` option several times.
+- `s, --source`: The data source to export. Only one can be specified from the command line.
+- `-r, \-\-region`: Set the region for which to export the data. The region is specified as a dictionary with the region type (here only `bbox` or `geom`) and the region coordinates.
+- `-t, \-\-time`: Set the time extent for which to export the data. The time extent is specified as a list with the start and end date.
+
+Here is an example of how to use the command:
+
+.. code-block:: console
+
+    > hydromt export /path/to/exported_data -d /path/to/data_catalog.yml -s "era5" -r "{'bbox': [4.68,53.19,4.69,53.20]}" -t "['2010-01-01', '2010-01-31']" -v
+
+If you want to export several sources or for more options, you can also use a configuration file instead.
+In that case, the main options are:
+
+- `-i, \-\-config`: Relative or absolute path to the export configuration file. The export configuration file is a yaml file that contains the sources, region, and time extent to export.
+
+And the command line would look like:
+
+.. code-block:: console
+
+    > hydromt export /path/to/exported_data -i /path/to/export_config.yml -v
+
+An example of the export file is:
+
+.. code-block:: yaml
+
+    export_data:
+        data_libs:
+            - /path/to/data_catalog.yml
+        region:
+            bbox: [4.68,53.19,4.69,53.20]
+        time_tuple: ['2010-01-01', '2020-12-31']
+        sources:
+            - hydro_lakes
+            - era5
+        unit_conversion: False
+        append: False
+        meta:
+            version: 0.1
+
+You can find detailed document on the function in `hydromt.DataCatalog.export_data <../_generated/hydromt.data_catalog.DataCatalog.export_data.rst>`_.
+For the region, only the `bbox` and `geom` types are supported, see the :ref:` region <region>` section for more information.
+
+Finally you can check the :ref:`hydromt export API <export_api>` for all the available options for the export command.
+
+Check command
+-------------
+The **hydromt check** command is used to validate the configuration file, the data catalog, and the region.
+It can be useful to validate files before running other command lines to avoid errors.
```

Comment:
A small addition to help manage expectations: 
```suggestion
It can be useful to validate files before running other command lines to avoid errors.
Please note that it will only check the syntax of the files provided. The actual data or calculations
referenced will not be checked, loaded or performed. 
```